Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_README.md

```diff
@@ -24,62 +26,63 @@ These task have the following names:
   - Acknowledge incident was received (task #13)
   - Update the user that the reported email is safe (task #16)
   - Update the user that the reported email is malicious (task #17
-  
+- Optional - Configure the `ExchangeLocation` input of the `Search And Delete Emails - Generic v2` playbook.
+- Optional - Personalize the inputs of the `Detect & Manage Phishing Campaigns` playbook.
+
 ##### Best Practices & Suggestions
 - The email received in the designated phishing inbox should be an email **containing** the potential phishing email as a file attachment, so that the headers of the original suspected email are retained.
 - Using Gmail or EWS v2 work best with the use case.
 - If phishing emails are forwarded instead of attached as files, Auto extract should not be turned off so that all indicators are properly extracted and analyzed.
 - Configuring the optional configurations can greatly enhance the investigation.
 
-
 ## Dependencies
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
-#### Sub-playbooks
-* Extract Indicators From File - Generic v2
-* Domain Enrichment - Generic v2
-* URL Enrichment - Generic v2
-* Block Indicators - Generic v2
-* Search And Delete Emails - Generic
+### Sub-playbooks
 * Calculate Severity - Generic v2
+* Email Address Enrichment - Generic v2.1
 * Process Email - Generic
-* File Enrichment - Generic v2
+* Extract Indicators From File - Generic v2
 * Detonate File - Generic
-* IP Enrichment - External - Generic v2
-* Email Address Enrichment - Generic v2.1
+* Entity Enrichment - Phishing v2
+* Search And Delete Emails - Generic v2
+* Block Indicators - Generic v2
+* Detect & Manage Phishing Campaigns
 
-#### Integrations
-* Builtin
+### Integrations
+This playbook does not use any integrations.
```

Comment:
are you sure? it's sub playbooks do.
I'm not saying that we need all the list though.

Reply:
The SDK apparently does not drill-down, anymore and doesn't write `* Builtin`.
I will add it back